packages/composables/src/hooks/useUser.ts

```diff
@@ -32,6 +34,24 @@ interface UseUser {
   getAddresses: () => Promise<CustomerAddress[]>;
 }
 
+interface RegisterCustomerParams {
+  firstName: string;
+  lastName: string;
+  password: string;
+  salutation: string;
+  street: string;
+  city: string;
+  zipcode: string;
+  country: string;
```

Comment:
Do we need both country and countryId?

Reply:
Maybe I would reduce the whole interface to fit only required fields for registration

Reply:
yes, for now, add only required ones. I guess countryId is enough. And please keep dates in single fields ;D

Reply:
I've taken it from here
<img width="1027" alt="Zrzut ekranu 2020-01-17 o 14 17 15" src="https://user-images.githubusercontent.com/12700136/72615218-79a28c80-3934-11ea-8f63-3bb0e5915457.png">